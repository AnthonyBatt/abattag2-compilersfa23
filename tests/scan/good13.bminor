// this is a comment and wont be read
/* this is a * comment too * it also / won't be read / */
"there were some unread comments at the top"
