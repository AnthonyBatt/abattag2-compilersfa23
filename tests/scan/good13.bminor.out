TOKEN_STRING_LITERAL      there were some unread comments at the top
